x-pack/plugins/session_view/public/components/ProcessTreeNode/styles.ts

```diff
@@ -33,11 +33,11 @@ export const useStyles = ({ depth, hasAlerts }: StylesDeps) => {
       color: colors.text,
     };
 
-    const searchHighlight: CSSObject = {
-      backgroundColor: colors.highlight,
-      color: colors.text,
-      borderRadius: border.radius.medium,
-    };
+    const searchHighlight = `
```

Comment:
Is there a reason why we want to change it back to template string?

Reply:
yeah, currently ProcessTreeNode is setting the searchHighlight style using innerHTML, so it's like directly in the DOM, outside React, and that is currently causing the object style not being properly interpreted because is not using the babel transformation for emotion. 

Think in the future we will get rid of the innerHTML set for the search, it does not worth to refactor right now because we still don't know for sure how the search will be, I think there's even a possibility of being handled by the server